.github/workflows/test.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10-dev", "pypy3"]
```

Comment:
This should be in a different PR that covers 3.10 support.

Reply:
I'm planning to do this soon. Emergencies keep coming up, so I haven't gotten to it, yet.